references.py

```diff
@@ -304,6 +304,7 @@ def is_folder(self) -> bool:
             shift_back_chain='ShiftBack.chain',
             shifted_control_region_interval='control_region_shifted.chrM.interval_list',
             non_control_region_interval='non_control_region.chrM.interval_list',
+            blacklist_sites='blacklist_sites.hg38.chrM.bed'
```

Comment:
> might consider listing that one here too to ensure it is recopied if we ever need to regenerate our cache bucket.

I thought as we are using rsync here it will be coping the full dir and the named files are just for direct referral?

Reply:
You're absolutely right — never mind then. (If I'd realised that on yesterday's PR, I probably would have asked whether `dict`/`amb`/etc actually need to be explicitly listed…)